### PR TITLE
Add view for `telemetry.origin_content_blocking`

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This view deduplicates aggregates created by the `prio_processor` DAG in Airflow. Since the DAG inserts rows directly into BigQuery, it is possible to have duplicate rows in a day if the job is rerun. Each row corresponds to the aggregate count of a blocked origin during a day. The most recent row by timestamp is returned in the view.

### CODE REVIEW

File: sql/telemetry/origin_content_blocking/view.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    SELECT
+      *,
+      ROW_NUMBER() OVER (
+        PARTITION BY
+          submission_date,
+          batch_id,
+          origin
+        ORDER BY
+          TIMESTAMP DESC
+      ) AS insert_rank
```
Comment (relud) [inline sql/telemetry/origin_content_blocking/view.sql]: how many rows does this dataset have? this method is known to have performance issues at high volume. it sounds like it's not much, but i want to clarify.

File: sql/telemetry/origin_content_blocking/view.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    FROM
+      `moz-fx-data-shared-prod.telemetry_derived.origin_content_blocking`
+  )
+WHERE
+  insert_rank = 1
```
Comment (relud) [inline sql/telemetry/origin_content_blocking/view.sql]: this is inconsistent with this pattern's use elsewhere in the repo. I think we should:
1. rename `insert_rank` as `_n`
2. make the select outer select clause `SELECT * EXCEPT(_n)`
3. move the inner query to a CTE

File: sql/telemetry/origin_content_blocking/view.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ORDER BY
+  submission_date DESC,
+  batch_id,
+  aggregate DESC
```
[the remark this answers is not in the document]
Comment (acmiyaguchi) [reply sql/telemetry/origin_content_blocking/view.sql]: The upside of this is that `SELECT * from telemetry.origin_content_blocking` will return a useful view of the data at first glance.

File: sql/telemetry/origin_content_blocking/view.sql (hunk 1 above)
Comment (acmiyaguchi) [reply sql/telemetry/origin_content_blocking/view.sql]: At the moment, it's 3m rows, or about 20mb scanned for this view.

**Pull request merged**